src/sentry/api/serializers/models/notification_setting.py

```diff
@@ -29,7 +29,7 @@ def get_attrs(
         """
         from sentry.models import NotificationSetting
 
-        actor_mapping = {target.actor_id: target for target in item_list}
+        actor_mapping = {target.id: target for target in item_list}
```

Comment:
`item_list` is a list of `User` or `Team` objects. We need to look up their `NotificationSetting` records by the `target.actor_id`.

Reply:
`item_list` is a list of `Actor` objects now, if it's supposed to be `Users` or `Teams` I'll track that down and fix.

Reply:
Oops you're right. I think then we should rename `target` to `actor` and update the docstring.